API Testing/README.md

```diff
@@ -1,59 +1,144 @@
 # Bruno API Testing
 
-This directory contains [Bruno](https://www.usebruno.com/) collections to test the Pillar 2 ecosystem.
+This directory contains [Bruno](https://www.usebruno.com/) collections for the Pillar 2 ecosystem.
+
+> **Important**: These collections are intended as developer and tester aids during active development. They provide example API calls and simple tests for manual verification. These tests are **not included in CI/CD** and may become out of sync with the codebase if not manually updated. The plan is to remove these collections before handover to live services.
```

Comment:
+1 for removing the tests from here in the long term. I see no reason these can't all be properly automated 🤞